week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,16 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var usdPrice = "$" + price * 1.4 * 1.1;
```

Comment:
That's a cool idea :)
But when you add 1% to something, it becomes 101% not 110%. How can you fix your function?